website/docs/language/providers/requirements.mdx

```diff
@@ -252,36 +252,36 @@ incompatibilities, and let the root module manage the maximum version.
 ## Built-in Providers
 
 Most providers are distributed separately as plugins, but there
-is one provider that is built into OpenTF itself. This provider enables the
+is one provider that is built into OpenTofu itself. This provider enables the
 [the `terraform_remote_state` data source](/docs/language/state/remote-state-data).
 
-Because this provider is built in to OpenTF, you don't need to declare it
+Because this provider is built in to OpenTofu, you don't need to declare it
 in the `required_providers` block in order to use its features. However, for
 consistency it _does_ have a special provider source address, which is
 `terraform.io/builtin/terraform`. This address may sometimes appear in
-OpenTF's error messages and other output in order to unambiguously refer
+OpenTofu's error messages and other output in order to unambiguously refer
 to the built-in provider, as opposed to a hypothetical third-party provider
-with the type name "opentf".
+with the type name "tofu".
```

Comment:
Not sure about this one

Reply:
Let's raise an issue for this one too. I think this is still `terraform`